### PR TITLE
Lesson 1.9

### PR DESCRIPTION
**Pull request by viniciuspietscher**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -40,18 +41,38 @@ function App() {
...
+          path='/'
+          element={
+            <>
+              <header style={{ textAlign: "center" }}>
+                <h1>Todo List</h1>
+              </header>
+              <hr />
+            </>
+          }
+        />
```
Comment (kwilson05) [inline src/App.js:56]: This attribute took me awhile to figure out 😆. Nice job! 💯

**Review by kwilson05 — APPROVED** [review]
Nice on to hooks! 🎖️

File: src/App.js (hunk 1 above)
Comment (viniciuspietscher) [reply src/App.js:56]: :laughing: prettier made it look so not pretty!

Comment (kwilson05) [reply src/App.js:56]: Oh so pretty 😆

**Pull request merged**